tests/test_suite-docker.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
+import argparse
+
+from ssg_test_suite import test_suite
+import ssg_test_suite.test_env
+
+
+def create_common_parser():
+    common_parser = argparse.ArgumentParser(add_help=False)
+    common_parser.set_defaults(test_env=None)
+    common_parser.add_argument("--base-image",
+                               dest="base_image",
+                               metavar="IMAGE",
```

Comment:
I think `--base-image` should be a required option, because it fails completely if this option is not set. Please make it required option or set an existing default name.